config/settings.yml

```diff
@@ -3,6 +3,7 @@ Default: &defaults
   port: 3000
   approot: "http://localhost:3000"
   copyright: Insert copyright statement here
+  allowAuthDummy: True # Uncomment to allow Auth Dummy login
```

Comment:
This should either be set to False, or should be overridden in Staging and Production settings below.